README.md

```
@@ -1,14 +1,615 @@
+
+---
 
 ## Contributing
 
-For information on how to contribute to this repository, please see the [contributing guidelines](CONTRIBUTING.md).
\ No newline at end of file
+For guidance on how to contribute to this project, see the [contributing guidelines](CONTRIBUTING.md).
+
+## Support, feedback and troubleshooting
+
+Please visit http://support.ably.com/ for access to our knowledge base and to ask for any assistance. You can also view
```

Markdownlint (0.35.0)

Bare URL used

(MD034, no-bare-urls)